hugo/content/citation/_index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: How to cite the Genome Portal and the data
+---
+
+In line with the principles of _FAIR_ and _Open Science_, we encourage the reuse and recognition of material made available on the Swedish Reference Genome Portal.
+On this page, you will find information about how to cite the portal when reusing and referencing the content. Please note that the information on the portal is
+updated continuously, therefore it is important to refer to specific versions (or to provide access dates) within citations.
+
+Below are instructions for how to cite different aspects of the Genome Portal. These include:
+
+- Citing the the Swedish Reference Genome Portal website
+
+- Citing a specific species entry in the Genome Portal
+
+- Citing the data in the Genome Portal
+
+- Citing the underlying code of the Genome Portal, including its dependency on JBrowse 2
+
+### 1. Citing the Swedish Reference Genome Portal website
+
+The Resource Identification Portal was created in support of the <a target="_blank" href="https://www.rrids.org/">Resource Identification Initiative</a>.
+It aims to promote the identification, discovery, and reuse of research resources. Research Resource Identifiers (**RRIDs**) are persistent and unique identifiers for
+referencing a research resource. For official guidance see the [SciCrunch page on RRID citations](https://scicrunch.org/resources/about/guidelines).
+
+The RRID for the Swedish Reference Genome Portal is [**XXXXXXXX**](https://XXXXXXXXXX).
+
+By citing the portal using the RRID, you will facilitate further reuse of the portal, enable us to track that activity, and allow others to easily find the _Summary Report_
+for usage of the Swedish Reference Genome Portal.
+
+**In-text citation (APA format)**:\
+Swedish Reference Genome Portal (SciLifeLab Data Centre; _version number_; RRID: XXXXXXXXX).
+
+**Reference list (APA format)**:\
+Swedish Reference Genome Portal (_access date_), SciLifeLab Data Centre. Version (version number). <https://genomes.scilifelab.se>, RRID:XXXXXXXX. [Species DOI].
+
+You will find the version number of the portal at XXXXXthe bottom of the footer on any pageXXXXXX, or on our
+<a target="_blank" href="https://github.com/ScilifelabDataCentre/genome-portal/">genome-portal repository</a> under 'releases'.
+
+### 2. Citing a specific species entry
+
+If you would like to refer to a specific species entry from the Genome Portal, you can do so by taking the Genome Portal citation (Section 1) and adding the species name and the
+DOI pointing to its associated metadata in SciLifeLab Data Repository:
```

Comment:
Should we provide a link to the data repository here perhaps ?

Reply:
Sure!